xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -609,4 +651,9 @@ def importobj(srcfile,srcdir,objtype,objnames=None,dryrun=None,version=None,upda
           
 
               
-                 
+def envlist():
+    print("#<%s> : %s#\n"%('Notice','refer the variables in the inventory file with format "{{variable name}}"'))
+    print('%-15s : %-100s'%('variable name','description')) 
```

Comment:
If using fix length to print, it might cut the string sometime.  And then you need an option to show all length.
And for the description, it might be mess the output, if it contains `\n`

I'm okay for now, but this need to be enhanced later.